lib/kubeclient/missing_kind_compatibility.rb

```diff
@@ -17,7 +17,7 @@ class MissingKindCompatibility
         'replicationcontrollers'     => 'ReplicationController',
         'resourcequotas'             => 'ResourceQuota',
         'secrets'                    => 'Secret',
-        'securitycontextconstraints' => 'SecurityContextConstraints',
+        'securitycontextconstraints' => 'SecurityContextConstraint',
```

Comment:
curling k8s discovery I get
$ curl -k -s     -H "Authorization: Bearer $OSH_TOKEN"     https://$OSH_HOST:8443/api/v1
```
    {
      "name": "securitycontextconstraints",
      "namespaced": false,
      "kind": "SecurityContextConstraints",
      "verbs": [
        "create",
        "delete",
        "deletecollection",
        "get",
        "list",
        "patch",
        "update",
        "watch"
      ],
      "shortNames": [
        "scc"
      ]
    },

```

The purpose of this is to imitate the discovery so I think this is what we need. Have you any problems caused by this?